### PR TITLE
Recipe for "ipywe"

### PR DESCRIPTION
**Pull request by yxqd**
https://github.com/scikit-beam/ipywe

### CODE REVIEW

**Comment by yxqd** [comment]
@conda-forge/staged-recipes could you please merge this PR?

**Comment by yxqd** [comment]
@ocefpaf Could you please look at this or forward it to someone who can merge this?

File: recipes/ipywe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+python:
+  - 2.7
+  - 3.5
+  - 3.6
```
Comment (ocefpaf) [inline recipes/ipywe/conda_build_config.yaml]: You don't need that file.

File: recipes/ipywe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set gitorg = "scikit-beam" %}
+{% set name = "ipywe" %}
+{% set version = "condaforge" %}
```
Comment (ocefpaf) [inline recipes/ipywe/meta.yaml]: The version must be a number. Not sure what you want to do here.

File: recipes/ipywe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{% set sha256 = "464c20e8e10dd68f1cd7f4c96b2dc2014733b47e37de9340ddd23238ee14818c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+#  url: https://github.com/{{ gitorg }}/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://github.com/{{ gitorg }}/{{ name }}/archive/conda-forge.tar.gz
```
Comment (ocefpaf) [inline recipes/ipywe/meta.yaml]: Please point it to a release, like https://github.com/scikit-beam/ipywe/archive/0.1.1.tar.gz or https://pypi.org/project/ipywe/0.1.1a1/

**Comment by yxqd** [comment]
Thanks, @ocefpaf. I updated the recipe (fixed version number). Please check.

**Comment by yxqd** [comment]
Thanks, @ocefpaf. Can this be merged now?

**Comment by yxqd** [comment]
@ocefpaf  Thanks!

**Pull request merged**